plantcv/plantcv/roi_objects.py

```diff
@@ -79,16 +77,13 @@ def roi_objects(img, roi_type, roi_contour, roi_hierarchy, object_contour, obj_h
         kept_obj = cv2.bitwise_not(kept)
         mask = np.copy(kept_obj)
         obj_area = cv2.countNonZero(kept_obj)
-        kept_cnt, hierarchy = cv2.findContours(kept_obj, cv2.RETR_TREE, cv2.CHAIN_APPROX_NONE)[-2:]
-        cv2.drawContours(ori_img, kept_cnt, -1, (0, 255, 0), -1, lineType=8, hierarchy=hierarchy)
+        kept_cnt, kept_hierarchy = cv2.findContours(kept_obj, cv2.RETR_TREE, cv2.CHAIN_APPROX_NONE)[-2:]
+        cv2.drawContours(ori_img, kept_cnt, -1, (0, 255, 0), -1, lineType=8, hierarchy=kept_hierarchy)
         cv2.drawContours(ori_img, roi_contour, -1, (255, 0, 0), params.line_thickness, lineType=8,
                          hierarchy=roi_hierarchy)
 
     # Find the largest contour if roi_type is set to 'largest'
     elif roi_type.upper() == 'LARGEST':
-        # Print warning statement about this feature
-        print(
-            "Warning: roi_type='largest' will only return the largest extreme outer contour. All child contours are left behind.")
 
```

Comment:
print("Warning: roi_type='largest' will only return the largest extreme outer contour and its child contours. All grandchild contours are left behind.")